tests/Toolkit/ATest.php

```diff
@@ -447,6 +447,26 @@ public function testFill()
 			'elephant',
 			'elephant'
 		], A::fill($array, 5, 'elephant'));
+
+		// Callable
+		$this->assertSame([
+			'miao',
+			'wuff',
+			'tweet',
+			'elephant',
+			'elephant',
+			'elephant'
+		], A::fill($array, 6, function () {
+			return 'elephant';
+		}));
+
+		// Callable with Closure
+		$this->assertSame([1, 2, 3], A::fill([], 3, function (int $i) {
+			return $i + 1;
+		}));
+
+		// callable with callable
+		$this->assertSame([false, true, false], A::fill([], 3, [\Kirby\Toolkit\V::class, 'accepted']));
```

Comment:
Please put rather a `use` statement on top of the file, not the full namespace classname inline.